src/main/java/com/olgaboiar/mint/RequestParser.java

```diff
@@ -6,8 +6,12 @@
     Request parse(List<String> incomingRequest) {
         String[] requestLineArr = incomingRequest.get(0).split(" ");
```

Comment:
Can you think of a better name than `requestLineArr`? Generally I try to avoid names like `Arr` that describes the data structure but not the business domain idea.

Reply:
renamed in https://github.com/olgaboiar/MintHTTPServer/pull/6/commits/39acfd8c01a36590f549c1985adc08a5dcbf86bf